stream/chunk_writer.js

```diff
@@ -48,7 +57,10 @@ ChunkWriter.prototype._transform = function _transform(value, encoding, callback
     var err = tup[0];
     var buffer = tup[1];
     if (err) {
-        callback(err);
+        callback(ChunkWriteError(err, {
```

Comment:
This stack will not be what you want it to be.

However if you changed your synchronous interface of `[err, value]` into a callback then the stack will be what you want it to be!